runtime/storage.go

```diff
@@ -232,6 +232,14 @@ func (s *Storage) Commit(inter *interpreter.Interpreter, commitContractUpdates b
 	deltas := s.PersistentSlabStorage.DeltasWithoutTempAddresses()
 	common.UseMemory(s.memoryGauge, common.NewAtreeEncodedSlabMemoryUsage(deltas))
 
+	size := s.PersistentSlabStorage.DeltasSizeWithoutTempAddresses()
+	if size > 0 {
+		inter.ReportComputation(common.ComputationKindEncodeValue, uint(size))
+
+		usage := common.NewBytesMemoryUsage(int(size))
+		common.UseMemory(s.memoryGauge, usage)
+	}
```

Comment:
Maybe move this above the introduction of `deltas`.

Love how simple the PR got! 👌